db/migrate/20241024204511_create_interfaces.rb

```diff
@@ -0,0 +1,10 @@
+class CreateInterfaces < ActiveRecord::Migration[7.0]
+  def change
+    create_table :interfaces do |t|
+      t.string :external_id
+      t.integer :interface_type, null: false
+
+      t.timestamps
+    end
```

Comment:
indices please:

- one index for the `external_id` column, with uniqueness constraint
- one index for the `type` column

Reply:
Done!